CMakeLists.txt

```diff
@@ -20,8 +34,11 @@ target_include_directories(meshio INTERFACE
 
 target_compile_features(meshio INTERFACE cxx_std_17)
 
-include(CTest)
-add_subdirectory(test)
+if((CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME OR MODERN_CMAKE_BUILD_TESTING)
+  AND
+  (MeshIO_BUILD_TESTS OR MeshIO_BUILD_COVERAGE))
+  add_subdirectory(test)
+endif()
```

Comment:
We shouldn't do this. The super-project has to decide if they want to run tests or not. Let's just keep the condition as following.

`MeshIO_BUILD_TESTS OR MeshIO_BUILD_COVERAGE`

Reply:
Done.